README.md

```diff
@@ -101,11 +101,11 @@ command line tools to your `PATH`:
 
 #### Ninja
 [Ninja](https://ninja-build.org) is the current recommended build system
-for building Swift and is the default configuration generated by CMake. If
-you're on macOS or don't install it as part of your Linux distribution, clone
-it next to the other projects and it will be bootstrapped automatically:
+for building Swift and is the default configuration generated by CMake. [Pre-built
+packages](https://github.com/ninja-build/ninja/wiki/Pre-built-Ninja-packages)
+are available for macOS and Linux distributions. You can also clone Ninja
+next to the other projects and it will be bootstrapped automatically:
 
-##### Build from source
```

Comment:
The line above, "You can also clone Ninja next to the other projects and it will be bootstrapped automatically:", already states that we are going to give instructions to build Ninja from source.

The line removed was also smaller than the next lines "Via SSH" and "Via HTTPS" when reading the README, which was quite strange.

As this is quite a subjective change, I can revert it if my arguments do not convince you.